app/views/early_career_payments/claims/_one_time_password.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="govuk-grid-row">
```

Comment:
We currently don't send HTML emails, presumably because of the inherent complications.

Why are we doing this now and does this need testing across email clients?

Reply:
@nomasprime I assumed that the emails not being HTML was because that was the requirement when the product was originally delivered. It might be to do with how Notify is used too. I have just been consistent as we only have `.text.erb` templates in the existing repo

Reply:
Sorry, I realise what the actually change is now. All good 👍